src/OrderFormAcceptanceTests.Steps/Steps/PreviewOrderSummary.cs

```diff
@@ -121,6 +121,7 @@ public void ThenTheTotalAnnualCostIsDisplayed()
             value.Should().NotBeNullOrEmpty();
         }
 
+        [Then(@"the Total cost of contract is displayed")]
         [Then(@"the total cost of ownership is displayed")]
```

Comment:
Do we still need the `Then the total cost of ownership is displayed` step? everything related to that should be changed to match the new step definition

Reply:
It's still called total cost of ownership on the front end, and in the first scenario that lands on the page and asserts all the elements are displayed. I think it's only the context of the current story that we're calling it Total cost of contract